build.gradle

```diff
@@ -11,6 +11,7 @@ apply plugin: 'java'
 dependencies {
     testCompile 'junit:junit-dep:4.10'
     testCompile 'org.mockito:mockito-core:1.8.5'
+    testCompile 'com.pushtorefresh.java-private-constructor-checker:checker:1.0.0'
```

Comment:
It's a small (only one class), **test** dependency that won't affect project. 

@benjchristensen what is your opinion?